### PR TITLE
185/Connect_youtube

### PR DESCRIPTION
**Pull request by farouqBenarous**
# Connect to Youtube #184

## Issue(s) Reference:
- Github card  https://github.com/dashdash/integrator/issues/184

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.dashdash</groupId>
+        <artifactId>integrator-pom</artifactId>
+        <version>1.0.2</version>
+    </parent>
+
+    <groupId>com.dashdash.integration</groupId>
+    <artifactId>integration-youtube</artifactId>
+    <version>0.0.1</version>
```
Comment (DBastos-mindera) [inline pom.xml:12]: Should be a SNAPSHOT version, right?

**Review by DBastos-mindera — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (farouqBenarous) [reply pom.xml:12]: Yeap true

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
+        <kotlin-logging.version>1.7.8</kotlin-logging.version>
+        <camel.version>3.3.0</camel.version>
+        <spring-boot.version>2.2.7.RELEASE</spring-boot.version>
+        <dd.instructions.version>2.17.3</dd.instructions.version>
+        <dd.utils.version>2.1.3</dd.utils.version>
+        <integrator-commons.version>1.0.19</integrator-commons.version>
+        <mockk.version>1.9.3</mockk.version>
+    </properties>
+
```
Comment (gustavocortarelli) [inline pom.xml:37]: there is a different structure that should be used on pom files that makes this file simpler. Check the facebookads as example

Comment (farouqBenarous) [reply pom.xml:37]: Yes this file was generate by new.  archtype

**Review by gustavocortarelli — APPROVED** [review]

**Review by DBastos-mindera — APPROVED** [review]

**Pull request merged**